frontend/src/app/(navfooter)/page.tsx

```diff
@@ -8,7 +8,7 @@ import WelcomeInfo from "./WelcomeInfo"
 export async function generateMetadata(): Promise<Metadata> {
     const title = "decomp.me"
 
-    const description = "A collaborative decompilation platform."
+    const description = "A collaborative reverse-engineering platform for working on decompilation projects with others."
```

Comment:
This is the old description, we shortened it yesterday 🙂

Reply:
Huh I had just pulled all the changes off of main so I don't know how that's there I'll fix it later when I get a chance